checklists/clgmap.php

```diff
@@ -3,6 +3,8 @@
 include_once($SERVER_ROOT.'/classes/ChecklistManager.php');
 if($LANG_TAG != 'en' && file_exists($SERVER_ROOT.'/content/lang/checklists/clgmap.' . $LANG_TAG . '.php')) include_once($SERVER_ROOT . '/content/lang/checklists/clgmap.' . $LANG_TAG . '.php');
 else include_once($SERVER_ROOT.'/content/lang/checklists/clgmap.en.php');
+if($LANG_TAG == 'en' || !file_exists($SERVER_ROOT.'/content/lang/header.' . $LANG_TAG . '.php')) include_once($SERVER_ROOT . '/content/lang/header.en.php');
```

Comment:
shouldn't this just be in the include file so that is a all in one package?

Reply:
I tried this in collections/map/index.php, and it doesn't seem to be working. I think it's because includes/minimal_header_template.php doesn't get included until the `<body>` of the page.

Reply:
(And by, "doesn't work", I mean gives developer warnings about tags that can't be translated).